website/docs/reference/widgets/select.md

```diff
@@ -60,6 +60,31 @@ These properties allow editing the Select widget. All these properties are prese
 | **Filterable**            | Makes the dropdown list filterable.                                                                                                                                               |
 | **Server Side Filtering** | Enables server-side filtering via an API / Query request. Use this property when the Select widget's Option data is being bound to an API / Query.                                        |
 
+#### Default selected value
+
+This property allows you to specify a default value that can be selected when the widget is first displayed. This can be useful if you want to pre-populate the widget with a specific value, or if you want to ensure that a certain option is selected by default.  
+
+The default selected value should be set to the value of the option from the options property. This value should match the value attribute of the selected option.
+
+For example, if your options property is an array of objects with a **label** and a **value** property, such as
```

Comment:
Wherever the name of the property is mentioned, keep it title case. eg: here Options property